.github/workflows/chart-testing.yaml

```diff
@@ -0,0 +1,65 @@
+name: Helm Charts CI
+on:
+  pull_request:
+    paths:
+      - "charts/**"
+      - ".github/workflows/**"
+
+  test_list_changed:
+    name: Test (list changed)
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.list_changed.outputs.changed }}
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      # used to fetch all history for all branches and tags
+      with:
+        fetch-depth: 0
+    - name: setup_environment
+      uses: ./.github/workflows/setup_environment
+    - name: Run chart-testing (list-changed)
+      id: list_changed
+      run: |
+        changed=$(ct list-changed --config chart-testing.yaml)
+        if [[ -n "$changed" ]]; then
+          echo "::set-output name=changed::true"
```

Comment:
Suggestion: GitHub has or is deprecating the set-output https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Could this be changed to use a similar notation:
"""
- name: Set output
run: echo "{name}={value}" >> $GITHUB_OUTPUT
"""

Reply:
Agreed, will update